apps/web/src/components/Profile/Feed.tsx

```diff
@@ -59,7 +59,9 @@ const Feed: FC<Props> = ({ profile, type }) => {
     skip: !profile?.id
   });
 
-  const publications = data?.publications?.items;
+  const publications = [
+    ...new Map(data?.publications?.items?.map((publication) => [publication['id'], publication])).values()
```

Comment:
Im guessing this should be validated from backend and not on frontend. Backend is not supposed to send same post twice? 

PS: If you were going to remove duplicates why not use `set` ?

Reply:
Yes makes sense to use ‘Set()’

Reply:
But have we validated backend is sending twice? I think some problem with apollo cache 🙇🏼

Reply:
@saurabhdoteth did you validate the root cause of the issue?

Reply:
yes, it seems like apollo is causing the issue because when I checked the network tab it returned the correct number of posts(without duplications) but when I console log'd `data` from `useProfileFeedQuery()` it contained twice the number of publications.

Reply:
This fixed in https://github.com/lensterxyz/lenster/pull/1395